sql/views.sql

```diff
@@ -69,6 +69,23 @@ CREATE OR REPLACE VIEW timescaledb_information.policy_stats as
     INNER JOIN _timescaledb_internal.bgw_job_stat js on j.id = js.job_id
   ORDER BY ht.schema_name, ht.table_name;
 
+-- view for background worker jobs
+CREATE OR REPLACE VIEW timescaledb_information.jobs AS
+SELECT j.id AS job_id,
+  j.application_name,
+  j.schedule_interval,
+  j.max_runtime,
+  j.max_retries,
+  j.retry_period,
+  j.proc_schema,
+  j.proc_name,
+  j.owner,
+  j.scheduled,
+  j.config,
+  js.next_start
```

Comment:
This column is the same as `next_scheduled_run` in `policy_stats`. I guess it makes sense to have in both views, but then it is good to use the same name, and `next_scheduled_run` is more meaningful name.

Reply:
this view has all the settings you can set with alter_job where it is named next_start

Reply:
Using different names is confusing. The other possibility is to change the name in `policy_stats` or use `next_scheduled_run` everywhere.

Reply:
I agree , the name should be consistent. Makes sense to keep next_start here since that is the name of the setting for alter_job and rename  policy_stats view column to next_start.

Reply:
The `alter_job` can be renamed too ;) @gayyappan was there any discussion around the name `next_scheduled_run`?
We can fix names later, so now it  can be `next_start` in both views as part of this PR. I will include both views in the discussion doc.

Reply:
@k-rus We finalized on next_start for alter_job when we discussed the job apis. We just need to use that consistently across all the views.